### ISSUE TITLE
Feature request: clear column button

### ISSUE THREAD

**Issue by lashman1985** [NicolasConstant/sengi#427]
Mastodon added the ability to clear the notifications column a while ago, and it would be nice to have that ability represented in Sengi as well. Ideally - right next to the settings button: 

[something like this](https://files.mastodon.social/media_attachments/files/108/205/990/640/925/531/original/7374233c523c0ca1.png)

and i'm not entirely sure if the API allows it, but it would be even better if something like that could be added for all columns

**Comment by lashmanRB** [comment]
so, ummm, any news about this one? :)

**Comment by NicolasConstant** [comment]
No-one seems to have found the time and/or interest to work on that. 😕

If you (or someone you can convince) want to work on it, feel free to ask for insights / resources / etc, I will be more than happy to give you some insights and help you to get started.
